### PR TITLE
Home about and roles first commit

### LINKED ISSUE #25: Home, About and Roles pages

### PR DESCRIPTION
**Pull request by zackweb-pro**

### CODE REVIEW

File: src/components/About/AboutSectionTwo.tsx [hunk 1]
```diff
@@ -1,60 +1,34 @@
...
+
+</motion.div>
+          <motion.div
+       initial={{ opacity: 0, y: 50 }}
+       animate={{ opacity: 1, y: 0 }}
+       exit={{ opacity: 0, y: -50 }}
+       transition={{ duration: .6 }}
+       style={{margin: "auto"}} >
+      
+                <SliderSem></SliderSem>
```
Comment (ahmedidrissi) [inline src/components/About/AboutSectionTwo.tsx]: You can use self closed tag here if you want `<SliderSem />`

File: src/components/Roles/Role.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+"use client";
+import { useState } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import { IconType } from 'react-icons';
+
+type JobCardProps = {
```
Comment (ahmedidrissi) [inline src/components/Roles/Role.tsx]: Can you add this type to types folder like the others pls

File: src/components/Common/SectionTitle.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ const SectionTitle = ({
   paragraph,
   width = "570px",
   center,
-  mb = "100px",
+  mb = "50px",
```
Comment (ahmedidrissi) [inline src/components/Common/SectionTitle.tsx]: This is a common file. Changes here will affect all pages!

**Review by ahmedidrissi — CHANGES_REQUESTED** [review]
I have one small suggestion. Can you follow the template architecture. I mean like this:

- Components:
image

- Types:
image

**Comment by ahmedidrissi** [comment]
Another suggestion, can you replace "Cancel" button with "X" (top right):
image

File: src/components/Common/SectionTitle.tsx (hunk 3 above)
Comment (zackweb-pro) [reply src/components/Common/SectionTitle.tsx]: I return it the original 100px

File: src/components/Roles/Role.tsx (hunk 2 above)
Comment (zackweb-pro) [reply src/components/Roles/Role.tsx]: Done!

File: src/components/About/AboutSectionTwo.tsx (hunk 1 above)
Comment (zackweb-pro) [reply src/components/About/AboutSectionTwo.tsx]: Okey just vs code auto complete and I think it's not a big deal HHHHHHH (but I change it though)

**Review by ahmedidrissi — APPROVED** [review]
Well done!

**Review by Cristal32 — APPROVED** [review]

**Pull request merged**